src/Aspire.ProjectTemplates/templates/aspire-apphost/.template.config/dotnetcli.host.json

```diff
@@ -8,14 +8,27 @@
         "longName": "no-restore",
         "shortName": ""
       },
-      "appHttpPort": {
+      "appHostHttpPort": {
```

Comment:
These keys hadn't been updated since we renamed the "App" project to "AppHost" so cleaned it up as part of this PR.